lib/coverband/reporters/web.rb

```diff
@@ -12,9 +12,18 @@ class Web
       attr_reader :request
 
       def initialize
+        require "coverband/reporters/web"
```

Comment:
do you manually require this file somewhere in your app / initializer or whatever?

`require "coverband/reporters/web"` as I removed it from the top level gem require nothing should load it without the override.

Reply:
```"coverband/reporters/web"``` is being manually required. So this is in Gemfile:

```
gem 'coverband', require: false
```

This is in routes.rb

```ruby
require 'coverband/reporters/web'
```

And this is in application.rb

```ruby
Bundler.require(*Rails.groups(:assets)) if defined?(Bundler)
require_relative './configatron.rb'

if configatron.coverband.enabled!
  require 'coverband'
end
```

Reply:
OK, yeah I don't think we want

```
This is in routes.rb

require 'coverband/reporters/web'
```

I don't think we want folks to require in routes anymore... I think we could make it work if they do by duplicating all the extra requires in the normal initialize

Reply:
normal as in the one in web not the new override.

Reply:
Yeah i can totally fix this for this specific installation by not having the includes in the routes but think it would save everyone headaches if we try to make it as backwards compatible as possible.